### ISSUE TITLE
Cannot load dtbo file

### ISSUE THREAD

**Issue by jvanbruegge** [pmezydlo/SPI_slave_driver_implementation#7]
When I try to execute `echo 'SPI0_slave' > /sys/devices/platform/bone_capemgr/slots`, I get the error `bash: echo: write error: Invalid argument`. A short look into dmesg shows this error: `__of_adjust_tree_phandle_references: Illegal property (size) 'fixup' @/__local_fixups__`

Do you know what this means? Was not able to find something useful

**Comment by ghost** [comment]
**i am currently working on this to  
just want to tell you that on 4.4.8-ti-r22
and 3.70 it faild to create device at the end**

in your case make sure you using the /usr/bin/dtc when creating the dtbo 
on your bbb device

**Comment by SuFizz** [comment]
Did you guys figure this out @naor2205 @jvanbruegge ? Stuck in the same place.. @pmezydlo had indicated that some kernel was going to natively support the SPI slave.. Any ideas?

**Comment by pmezydlo** [comment]
A long time ago I don't see any progress in enabling slave mode. You can add debug message to my drivers code and see if it have any problem with loading dtbo. But I didn't see similar problem with compiling dtbo file.

**Comment by jvanbruegge** [comment]
No, I used a small atmel uC between the two systems that has two spi clients and copies over the data
